pynetbox_data_uploader/lib/utils/csv_to_dataclass.py

```diff
@@ -0,0 +1,41 @@
+import csv
+from typing import List
+from lib.utils.dataclass_data import Device
+
+
+def open_file(file_path: str) -> csv.DictReader:

```

Comment:
Whenever you're using file paths - you should use Pathlib to validate the filepath before you use it 

https://docs.python.org/3/library/pathlib.html